config/config_test.go

```diff
@@ -250,7 +250,7 @@ func TestLoadConfig(t *testing.T) {
 						NetworksOrGroups: []string{"127.0.0.1"},
 						TimeoutCfg: TimeoutCfg{
 							ReadTimeout:  Duration(time.Minute),
-							WriteTimeout: Duration(90 * time.Second),
+							WriteTimeout: Duration(3 * time.Minute),
```

Comment:
why also this change?

Reply:
because (I don't know why) write timeout = max_execution_time + 1 minute, cf
https://github.com/ContentSquare/chproxy/blob/master/config/config.go